package.json

```
@@ -6,9 +6,9 @@
    "license": "MIT",
    "scripts": {
       "test": "echo \"Error: no test specified\"",
-      "dev": "nodemon src/index.js",
-      "lint": "eslint src/**/*.js",
-      "lint:fix": "eslint src/**/*.js --fix",
+      "dev": "nodemon usuario-services/src/index.js",
+      "lint": "eslint usuario-services/src/**/*.js",
+      "lint:fix": "eslint usuario-services/src/**/*.js --fix",
```

```suggestion
      "dev": "nodemon ./src/index.js",
      "lint": "eslint./src/**/*.js",
      "lint:fix": "eslint ./src/**/*.js --fix",
```

Acho que aqui pode usar um caminho relativo, vale a pena fazer um teste disso pq não tenho certeza